src/test/java/io/confluent/connect/elasticsearch/ElasticsearchWriterTest.java

```diff
@@ -379,7 +361,7 @@ public void testIneffectiveDelete() throws Exception {
 
     ElasticsearchWriter writer = initWriter(client, BehaviorOnNullValues.DELETE);
     writeDataAndRefresh(writer, records);
-    verifySearchResults(new ArrayList<SinkRecord>());
+    verifySearchResults(new ArrayList<>());
```

Comment:
Any particular reason we changed this?

Reply:
just cleaning up unnecessary types since that method accepts `Collection<SinkRecord>`